content/en/api/payments.md

```diff
@@ -5,4 +5,99 @@ position: 41
 category: API
 ---
 
-Payments
\ No newline at end of file
+## Capture
+
+The capture endpoint allows capturing funds from a Single Payment authorisation, a Frequent Payment or a Frequent Payment authorisation. It also allows getting the details of a previous capture.
+
+### Capture payment
+
+`POST` `/capture/<payment id>`
+
+#### Parameters
+
+<div class="ep-protocol-list">
+
+  - `transaction_key`: `string` (<= 50 characters) with the internal key used to identify this transaction in the merchant's database.
+  - `capture_date`: `string` in the format `'Y-m-d'` with the capture date.
```

Comment:
The format should be `YYYY-mm-dd`, no?

Reply:
I believe the format follows the PHP conventions (https://www.php.net/manual/en/datetime.format.php). YYYY would repeat the year 4 times. Original formats were all copied from the current docs: https://api.prod.easypay.pt/docs#

Reply:
It doesn't repeat the year 4 times, it formats the date with `4-digit year`, 2-digit month (with leading zeros), 2-digit day (with leading zeros). Ex: 2022-08-04. This is the format used in the docs:

https://api.prod.easypay.pt/docs#tag/Payment-Generic-Operations/paths/~1capture~1%7Bid%7D/post
> "capture_date": "2017-11-11",

Reply:
You're misunderstanding. Y is already the 4 characters in the specification. YYYY would be the year repeated 4 times. That's why the documentation already has Y-m-d (notice also the difference in uppercase/lowercase).

Reply:
I hate PHP formats. Legacy prog language!